client/web-sveltekit/src/routes/[...repo=reporev]/(validrev)/(code)/-/blob/[...path]/+page.svelte

```
@@ -102,6 +103,9 @@
     <FileHeader>
         <FileIcon slot="icon" file={blob} inline />
         <svelte:fragment slot="actions">
+            {#await data.externalServiceType then externalServiceType}
```

My assumption is that we don't need to display an error message. Firstly I expect much more to break when we can't resolve the repo info, and secondly an error message in place of the editor button doesn't leave a lot of space to inform users what they should do. In that case I feel like just dropping the feature until the errors resolve should be fine.

---

Will this log an error to the console? Ideally, we shouldn't just silently drop errors or we might just never know we broke something

---

I've updated this so it won't swallow the error anymore. I opted for a very prominent alert, as an error here indicates that we broke an API.